client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/dto/database/schema/CompositeTypeColumnDto.java

```diff
@@ -0,0 +1,40 @@
+package org.evomaster.client.java.controller.api.dto.database.schema;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CompositeTypeColumnDto {
+
+    /**
+     * The name of the composite type
+     */
+    public String name;
+
+    /**
+     * Returns if the column is a composite or a built-in type
+     */
+    public boolean isCompositeType;
+
+    /**
+     * The name of the composite type
+     */
+    public String type;
+
+    public int size;
+
+    public boolean nullable;
+
+    public boolean isUnsigned = false;
```

Comment:
why this redundant `= false`?  and, even if having it, shouldn't as well `nullable` have it to make it consistent?

Reply:
I will remove the =false to keep consistence.